### PR TITLE
chore: add commit zEn and

### PR DESCRIPTION
**Pull request by marianzburlea**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "extends": [
+    "airbnb",
```
Comment (QuintyHH) [inline .eslintrc.json:3]: Nice :)

**Review by Constantin-Matei — COMMENTED** [review]
I see :)

**Review by marianzburlea — COMMENTED** [review]
nu pot sa-mi accept singur

**Review by QuintyHH — APPROVED** [review]
:100:

**Review by robertoalx — APPROVED** [review]
Accepted :D

**Review by Constantin-Matei — APPROVED** [review]

**Pull request merged**